src/WebJobs.Extensions/Extensions/Warmup/Trigger/WarmupTriggerListener.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Listeners;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Warmup.Trigger
+{
+    internal class WarmupTriggerListener : IListener
```

Comment:
Rename to NullListener, and move to a private class in WarmupTriggerBinding to bury it